src/modm/platform/i2c/stm32-extended/i2c_shared_irqs.cpp.in

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
This file will generated empty if you only use I2C1 -> Is there an option to not generate at all if not needed?

Reply:
See the comment above. The file won't be generated anymore, if there are no shared interrupts with that change.